scripts/docker-entrypoint-charlie.sh

```diff
@@ -12,7 +12,7 @@ echo "Chain Version is ${CHAIN_VERSION}"
   --base-path /tmp/polkadot-chains/charlie \
   --bootnodes /ip4/${ENDPOINT_DEVELOPMENT}/tcp/30333/p2p/${BOOTNODE_NODE_ID_LOCAL} \
   --keystore-path "/tmp/polkadot-chains/charlie/keys" \
-  --chain /dhx/node/node/src/chain-built/chain_def_${CHAIN_VERSION}.json \
+  --chain ${CHAIN_VERSION} \
   --charlie \
   --name "${NODE_ENV} Validator Charlie" \
   --port 30335 \
```

Comment:
if Alice, Bob, Charlie, Dave, and Even are all run in different Docker containers then can't we also make these all as follows (like you've done for Harbour chain nodes since they'd also all be on different Docker containers). As the related docker-compose file exposes these container ports on the server with different ports so there aren't any conflicts:
```
  --port 30333 \
  --ws-port 9944 \
  --rpc-port 9933 \
```